packages/web/src/components/table/Table.tsx

```diff
@@ -377,11 +377,8 @@ export const Table = ({
           )}
           {...props}
           key={key}
-          onClick={
-            isLocked
-              ? undefined
-              : (e: MouseEvent<HTMLTableRowElement>) =>
-                  onClickRow?.(e, row, row.index)
+          onClick={(e: MouseEvent<HTMLTableRowElement>) =>
```

Comment:
will this mean we still register clicks even if the row has disabled style? or is that supposed to be handled in `onClickRow`?

Reply:
i added this recently the undefined logic at the left but we dont need it. the callers handle it. good call on the disabled style thing but i dont think we care atm